api/openapi.yaml

```diff
@@ -30,228 +30,654 @@ tags:
+      operationId: get-users
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                properties:
+                  users:
+                    items:
+                      $ref: ./model/User.v1.yaml
+                    type: array
+                type: object
+          description: Ok.
         '401':
           $ref: '#/components/responses/Unauthorized'
         '403':
           $ref: '#/components/responses/Forbidden'
         default:
           $ref: '#/components/responses/DefaultResponse'
-      summary: Create a place in an organization
+      summary: Read all users.
+      tags:
+        - stalker web app
+    post:
+      description: Create a new user.
+      operationId: post-users
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                loginData:
+                  $ref: ./model/LoginData.v1.yaml
+                userData:
+                  $ref: ./model/UserData.v1.yaml
+              required:
+                - loginData
+                - userData
+              type: object
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: ./model/GenericId.v1.yaml
+          description: OK
+        '400':
+          content:
+            application/json:
+              schema:
+                properties:
+                  message:
+                    enum:
+                      - This email is not valid or is already registered
+                    type: string
+                required:
+                  - message
+                type: object
+          description: Bad Request
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      security: []
+      summary: Create a user.
+      tags:
+        - stalker mobile app
+  /version:
+    get:
+      description: Get the current server version.
+      operationId: get-version
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  version:
+                    type: string
+                    description: "The server's version number."
+                    example: 0.1.0
+                    pattern: "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
+                required:
+                  - version
+          description: Ok. The response contains the current server version.
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      summary: "Read the server's current version."
+      tags:
+        - stalker web app
+        - stalker mobile app
+      security: []
+  '/organization/{organizationId}/report':
+    parameters:
+      - $ref: '#/components/parameters/organizationId'
+    get:
+      summary: Read an organizational report.
```

Comment:
organizational?

Reply:
si, vuol dire "relative to an organization". è leggermente diverso dall'equivalente italiano (organizzativo), nel senso che "organizational report" non si traduce in "report organizzativo" ma in "report dell'organizzazione"